lib/udf_json_dicts.rb

```diff
@@ -0,0 +1,46 @@
+class UdfJsonDicts
+  UDFS = [
+      {
+          type:        :function,
+          name:        :json_extract_path_keys,
+          description: "Return the list of keys in a json dict as a json string",
+          params:      "jsonstr varchar(max)",
+          return_type: "varchar(max)",
+          body:        %~
+            import json
+            if not jsonstr:
+              return '[]'
+            else:
+              return json.dumps(sorted(json.loads(str(jsonstr)).keys()))
```

Comment:
Would it be a really heavy operation to sort these? will it be pushed down to redshift?